src/main/java/core/basesyntax/ColorSupplier.java

```diff
@@ -1,7 +1,10 @@
 package core.basesyntax;
 
+import java.util.Random;
+
 public class ColorSupplier {
-    public String getRandomColor() {
-        return null;
+    public Colour getRandomColor() {
+        int index = new Random().nextInt(Colour.values().length);
+        return Colour.values()[index];
     }
```

Comment:
Creating a new instance of Random every time the method is called can be inefficient. Instead, you can create a single instance as a class member and reuse it.

Reply:
Fixed